recipe/meta.yaml

```diff
@@ -30,8 +30,11 @@ requirements:
     - zlib
 
 test:
+  requires:
+    - make
    commands:
      - make_watertight --help
+     - make test
```

Comment:
This doesn't work because these tests must be performed on the as-installed code and not in the source tree.  The source tree containing DAGMC tests is gone by the time we get here.

Reply:
If you see my first commit - it did this and has now undone it...

Reply:
sorry